### PR TITLE
About, Reading, Reinforcement Database Driven

### PR DESCRIPTION
**Pull request by Mster115**
- [x] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [x] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [x] No dead links
- [x] Navbar is consistent across the site
- [x] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [x] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/assets/stylesheets/pages/CaseStudy.scss [hunk 1]
```diff
@@ -51,13 +51,16 @@
   > div {
     font-size: 0.9rem;
     display: flex;
-    flex-direction: column;
+    // flex-direction: column;
```
Comment (AndreasLc1103) [inline client/src/assets/stylesheets/pages/CaseStudy.scss]: remove

File: client/src/components/body/Reading/NonBulletList.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * list like view of information to the reading section of the lab.
+ */
+const NonBulletList = ({data}) => {
+    return(
+        <>
+        <ul className="non-bullet-list">
+            {data.map((text , index) =>{
+                return(
+                    <li key={index}>
+
```
Comment (AndreasLc1103) [inline client/src/components/body/Reading/NonBulletList.jsx]: newline not needed

File: client/src/components/body/Reading/NonBulletList.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const NonBulletList = ({data}) => {
+    return(
+        <>
+        <ul className="non-bullet-list">
+            {data.map((text , index) =>{
+                return(
+                    <li key={index}>
+                        
+                            <h5>{text.header}</h5>
+
```
Comment (AndreasLc1103) [inline client/src/components/body/Reading/NonBulletList.jsx]: same thing here

File: client/src/components/body/lab/LabGeneration.js [hunk 4]
```diff
@@ -37,6 +38,7 @@ const LabGeneration = (props)=>{
   });
 
   if (labInformation !== null && labInformation.length > 0 && progressState){
+      console.log("poop")
```
Comment (AndreasLc1103) [inline client/src/components/body/lab/LabGeneration.js]: I want to say leave this but remove

File: client/src/assets/stylesheets/pages/CaseStudy.scss (hunk 1 above)
Comment (AndreasLc1103) [reply client/src/assets/stylesheets/pages/CaseStudy.scss]: Appear

**Review by AndreasLc1103 — CHANGES_REQUESTED** [review]

File: client/src/components/body/landingpage/citation/ProfileGeneration.js [hunk 5]
```diff
@@ -21,120 +24,53 @@ function renderProfileData(profileInformation) {
         );
     })
   }
+  
+  const ProfileGeneration = (props) =>{
+    //const {}=props;
```
Comment (AndreasLc1103) [inline client/src/components/body/landingpage/citation/ProfileGeneration.js]: remove

File: client/src/components/body/landingpage/citation/SlideSet.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                return (
+                  <CarouselItem
+                    onExiting={onExiting}
+                    onExited={onExited}
+                  >
+                   {item}
+                  </CarouselItem>
+                );
+              }))
+        // eslint-disable-next-line
```
Comment (AndreasLc1103) [inline client/src/components/body/landingpage/citation/SlideSet.js]: do we need this?

**Review by Stormkiller-01 — COMMENTED** [review]
- Am now understanding as to why the lab1 - lab5 about, reading, and reinforcement files are now unneeded since by making the project more db driven, having general files that can encompass the settings needed for all 5 labs, via less files overall.

**Review by AndreasLc1103 — APPROVED** [review]

**Pull request merged**